zcash_extensions/src/transparent/demo.rs

```diff
@@ -478,16 +478,14 @@ impl<'a, B: ExtensionTxBuilder<'a>> DemoBuilder<B> {
 mod tests {
     use blake2b_simd::Params;
     use ff::Field;
+    use incrementalmerkletree::{frontier::CommitmentTree, witness::IncrementalWitness};
```

Comment:
```suggestion
```

Reply:
I think this can also be replaced by `sapling::{CommitmentTree, IncrementalWitness}`.